Tasks/Maven/mavenutil.ts

```diff
@@ -291,9 +292,10 @@ export function getExecOptions(): tr.IExecOptions {
     };
 }
 
-export function publishMavenInfo(mavenInfo:string) {
-    let stagingDir: string = path.join(os.tmpdir(), '.mavenInfo');
-    let infoFilePath: string = path.join(stagingDir, 'MavenInfo.md');
+export function publishMavenInfo(mavenInfo: string) {
+    const stagingDir: string = path.join(os.tmpdir(), '.mavenInfo');
+    const randomString: string = uuidV4();
+    const infoFilePath: string = path.join(stagingDir, 'MavenInfo-' + randomString + '.md');
```

Comment:
Does this file name show up in the UI anywhere - section header?

Reply:
No, not that I see.  The contents of the file appear in build summary beneath a "Maven" header; I don't see the file itself or its name anywhere.

Reply:
You may want to log the name of the file, so if someone is debugging an issue it's in the logs.

Reply:
Personally, I'm not sure it's needed.  If you look below, you can see the log '##vso[task.debug][Maven] Uploading build maven info from /tmp/.mavenInfo/MavenInfo-'.  I think that may be enough (though it is relying on the agent to log it).